Superbuild/External_GLEW.cmake

```diff
@@ -0,0 +1,55 @@
+############
+# This code was adapted from https://github.com/Slicer/Slicer/blob/main/SuperBuild/External_zlib.cmake
+############
```

Comment:
Specify this in the commit message indicating the version of Slicer by using `Slicer/Slicer@abcdefghi` where `abcdefghi` are the first 9 characters of the corresponding SHA